pkg/manager/manager_test.go

```diff
@@ -0,0 +1,265 @@
+package manager
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/docker/infrakit/pkg/discovery"
+	leader_file "github.com/docker/infrakit/pkg/leader/file"
+	group_mock "github.com/docker/infrakit/pkg/mock/spi/group"
+	store_mock "github.com/docker/infrakit/pkg/mock/store"
+	group_rpc "github.com/docker/infrakit/pkg/rpc/group"
+	"github.com/docker/infrakit/pkg/rpc/server"
+	"github.com/docker/infrakit/pkg/spi/group"
+	_ "github.com/docker/infrakit/pkg/store"
```

Comment:
What's the purpose of the underscore import?  If it remains, can you leave a comment so it's clear why it needs to be here?